### PR TITLE
ADDING AI CONTROL TO THE APPLICATION

### PR DESCRIPTION
**Pull request by Mutti499**
I think I made progress and I am opening it just to let you know this may be ready but other than that It should not be merged just yet.

### CODE REVIEW

**Comment by ErayEroglu** [comment]
We can add a feature which checks code snippets for malicious actions. If you think AI can be integrated for this task, maybe we can work on it. I can also contribute the implementation part

File: django_project_491/django_app/views/question_views.py [hunk 1]
```diff
@@ -143,9 +144,19 @@ def create_question(request: HttpRequest) -> HttpResponse:
             Lang2ID = get_languages() 
             language_id = Lang2ID.get(language, None)
 
+
             if language_id is None:
                 return JsonResponse({'error': 'Invalid language'}, status=400)
-        
+            
+            if (random() < 0.5):
```
Comment (serhan-cakmak) [inline django_project_491/django_app/views/question_views.py:151]: I didn't understand why you added this line

**Comment by serhan-cakmak** [comment]
After the modification I pointed out is made, this pr can be merged. I added the test I used below.       
image

File: django_project_491/django_app/views/question_views.py (hunk 1 above)
Comment (serhan-cakmak) [inline django_project_491/django_app/views/question_views.py]: I deleted this line in my commit. If there is a reasonable answer to this, I am all ears.

**Comment by Mutti499** [comment]
> We can add a feature which checks code snippets for malicious actions. If you think AI can be integrated for this task, maybe we can work on it. I can also contribute the implementation part

We figured that out in our second latest commit. Nice catch! Merging the PR

**Pull request merged**